content/_layouts/base.njk

```diff
@@ -7,6 +7,7 @@
   data-ccs-colors>
   <head>
     <meta charset="utf-8" />
+    <meta name="theme-color" content="#4285f4" />
```

Comment:
I'm curious where this color comes from?

Reply:
Removed. 🤦